code/game/objects/items/weapons/melee/energy.dm

```diff
@@ -103,6 +194,11 @@
 	attack_verb = list("attacked", "chopped", "cleaved", "torn", "cut")
 	sharp = 1
 	edge = 1
+	base_reflectchance = 0
+	base_block_chance = 0 //cannot be used to block guns
+	shield_power = 0
+	can_block_bullets = 0
+	armor_penetration = 80

```

Comment:
these armour penetration values are insane too. 80 is the highest melee armoour you can usually get. This entirely negates armor. You're removing all reason to even bother with riot armor

Theres no cause for armor pen values to be this high. Even a value as low as 15 is pretty powerful, 30 is pushing it, that much essentially cripples armor. 

This is basically removing a mechanic completely. have you no sense of balance?